mad-dog/src/tests/test_snapbackSM.js

```diff
@@ -82,34 +82,34 @@ const snapbackSMParallelSyncTest = async ({
     }
   }
 
-  // Issue parallel uploads for all users
-  await executeAll(async (libs, i) => {
-    // Retrieve user id if known from walletIndexToUserIdMap
-    // NOTE - It might be easier to just create a map of wallets instead of using 'index'
-    const userId = walletIndexToUserIdMap[i]
-    const newTrackMetadata = getRandomTrackMetadata(userId)
-    const randomTrackFilePath = await getRandomTrackFilePath(TEMP_STORAGE_PATH)
-    logger.info(
-      `Uploading Track for userId:${userId} (${libs.walletAddress}), ${randomTrackFilePath}, ${JSON.stringify(newTrackMetadata)}`
-    )
-    try {
-      const startTime = Date.now()
-      const trackId = await executeOne(i, (l) =>
-        uploadTrack(
-          l,
-          newTrackMetadata,
-          randomTrackFilePath
-        )
-      )
-      const duration = Date.now() - startTime
-      logger.info(`Uploaded track for userId=${userId}, trackId=${trackId} in ${duration}ms`)
-    } catch (e) {
-      logger.error(`Error uploading track for userId:${userId} :${e}`)
-    }
-  })
+  // // Issue parallel uploads for all users
```

Comment:
I think there might be some value to having a mode to just create x users, let's chat about this independently

Reply:
seed users via service-commands